src/layouts/main/styles/functions.js

```diff
@@ -0,0 +1,74 @@
+import {css} from 'styled-components';
+import {drawerWidth} from './constants';
+
+/**
+ * To return drawer styles when is open.
+ *
+ * @param {object} theme
+ * @return {css} drawer styles when opened
+ */
+const getOpenDrawerStyles = (theme) => css`
+  width: ${drawerWidth}px;
+  transition: ${theme.transitions.create('width', {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.enteringScreen,
+  })};
+`;
+
+/**
+ * To return drawer styles when is closed.
+ *
+ * @param {object} theme
+ * @return {css} drawer styles when closed
+ */
+const getClosedDrawerStyles = (theme) => css`
+  transition: ${theme.transitions.create('width', {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.leavingScreen,
+  })};
+  overflow-x: hidden;
+  width: ${theme.spacing(9) + 1}px;
+
+  .MuiListItemText-root {
+    visibility: hidden;
+  }
+`;
+
+/**
+ * To return drawer styles according to
+ * the opened state
+ *
+ * @param {{open: boolean, theme: object}} props
+ * @return {css} styles according to open state
+ */
+const drawerToggles = ({open, theme}) => {
```

Comment:
Maybe name this toggleDrawer instead? Tend to like the verb since we're doing something with this function.

Reply:
I decided to name it `getToggleDrawerStyles`. I think `toggleDrawer` sounds more like a function to handle component functionality. I just don't want to create confusion with the purpose of these functions.

Let me know if it sounds better, I was thinking about it for a few minutes